entrypoint/microservice/routes/documents.py

```diff
@@ -0,0 +1,219 @@
+# All routes related to /documents endpoint
+# We will provide the following routes:
+#   /documents
+#   /documents/id
+#   /documents/id/similar
+#   /documents/id/similarity_update
+#   /documents/retrieve
+
+import sys
+import requests
+from flask import (
+    Blueprint, flash, g, redirect, request, session, url_for, jsonify, current_app as app
+)
+from ..config import config_db
+from werkzeug.exceptions import abort
+
+bp = Blueprint('documents', __name__, url_prefix='/api/v1/documents')
+
+def get_documents_from_db(document_ids):
+    """
+    Function receives a list of document ids and returns a list of dictionaries of documents data.
+
+    Parameters:
+        documents_ids : list(int)
+            list of document ids
+    
+    Returns:
+        success (boolean), list of dictionaries of document data if the extraction from the database was successful.
+    """
+
+    db = config_db.get_db()
+    if db.cursor is None:
+        return False, {'Error' : 'The connection could not be established'}
+    
+    statement = "SELECT * FROM documents WHERE document_id IN %s;"
+    try:
+        db.cursor.execute(statement, (tuple(document_ids),))
+    except:
+        return False, {'Error' : 'You provided invalid document ids.'}
+    
+    # Enumerating the fields
```

Comment:
Lines 42, 43, and 44 should be handled in `library/postgresql.py` file (in accordance with the previous comment).